bindings_test.go

```diff
@@ -40,7 +40,7 @@ func TestBindingFloat64(t *testing.T) {
 					dbt.Errorf("%s: %g != %g", v, expected, out)
 				}
 			} else {
-				dbt.Fatalf("%s: no data", v)
+				dbt.Errorf("%s: no data", v)
```

Comment:
The change on the right is the same with upstream code.

The change on the left seems to be artificially introduced in the "merged commit":
https://github.com/observeinc/gosnowflake/commit/c4d7bfe0bfcf0a9712e731a4ef24972a8d054c43#diff-89d7a56d38f9f263a0c8f47053c091b9808cb45e9ac35f834275a860e8f9b2fc

We do have one Observe commit changing some tests, but AFAICT these functions are not included in them: https://github.com/observeinc/gosnowflake/commit/52d1540a67c2ca96d2be3987ab46059569f47cf6